code/src/components/CompleteList.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { todos } from 'reducers/todos'
+
+import './style/Listed.css'
+
+export const CompleteList = ({ todoIndex }) => {
+  const dispatch = useDispatch()
+  const todo = useSelector((store) => store.todos.list.todos[todoIndex])
```

Comment:
If you pass the whole todo item as a prop instead of only the index, you don't need to get the specific todo in this component again.